docs/content/preview/architecture/docdb-replication/async-replication.md

```diff
@@ -43,7 +43,7 @@ Multiple flows can be used; for example, two unidirectional flows between two un
 
 Although for simplicity, we will describe flows between entire universes, flows are actually composed of streams between pairs of tables, one in each universe, allowing replication of only certain namespaces or tables.
 
-xCluster is more flexible than a hypothetical scheme whereby read replicas are promoted to full replicas when primary replicas are lost because it does not require the two universes to be tightly coupled.  With xCluster, for example, the two universes can be running different versions of YugabyteDB and the same table can be split into tablets in different ways in the two universes.  xCluster also allows for bidirectional replication, which is not possible using read replicas because read replicas cannot take writes.
+xCluster is more flexible than a hypothetical scheme whereby read replicas are promoted to full replicas when primary replicas are lost because it does not require the two universes to be tightly coupled. With xCluster, for example, the same table can be split into tablets in different ways in the two universes and (in the future) the two universes will be able to run different versions of YugabyteDB. xCluster also allows for bidirectional replication, which is not possible using read replicas because read replicas cannot take writes.
```

Comment:
> the two universes will be able to run different versions of YugabyteDB

That is not true. And we dont know if we will ever support it

Reply:
Removing that part.